lib/convert.js

```diff
@@ -58,6 +58,9 @@ module.exports = function convert(config) {
         loc: nodeUtils.getLoc(node, ast)
     };
 
+    nodeUtils.map.set(result, node);
```

Comment:
I think these should come right before we return the result, that way you don't need to have two lots of them (i.e. there is no need to have them again in node-utils)

Reply:
`map` and `reverseMap` are not clear names IMO

Reply:
I set them when the nodes are created so that in e.g.
```ts
export class Foo {}
```
the `ClassDeclaration` and `ExportNamedDeclaration` have a TS AST node attached to them (`ClassDeclaration`).